### ISSUE TITLE
thumbnail files are not generated in the .ts folder in the Pro version while using ownCloud sync

### ISSUE THREAD

**Issue by joephein** [tagspaces/tagspaces#450]
I found out that the thumbnails were only generated for one folder. For any other or new folders tagspaces only creates the .ts folder with the .json and .txt files. I accidentally deleted those created thumbnails using the owncloud sync client and now I do not have any generated thumbnails at all. I have the "Enable thumbnails generation" enabled in the settings. I also tried to use the "update thumbnails" using the right mouse-click context menu on a folder but it has no effect. I am using tagspaces Pro 2.3.0 64 bit on Windows 10.

**Comment by joephein** [comment]
I want to upload the generated thumbnails to my owncloud so they can be hopefully used in the hosted-edition. Currently the on-the-fly generation of thumbnails in the hosted-edition is taking very long in the hosted edition. I hope that similar like the sidecar tags in the .ts folder the lower-res thumbnail can be used as well.

**Comment by uggrock** [comment]
Uploading the thumbs to you webdav server should work.

**Comment by joephein** [comment]
The issue is that the thumbnails are not created in the tagspaces Pro 2.3.0 64 bit on Windows 8.1 enterprise. Is there a way in the UI to have these thumbnails generated and stored in the .ts folder?

**Comment by uggrock** [comment]
Do you still have issue the thumbs generation? If yes try to disable the ownCloud sync for this folder.

**Comment by joephein** [comment]
I have tried that but it did not made a difference. The .txt and .json files are generated well though.

![owncloud_tagspaces](https://cloud.githubusercontent.com/assets/12627719/15966930/8e43ccde-2f25-11e6-9bfb-8df8fe958efe.PNG)

I have another screenshot of the sync status in Windows explorer:

![owncloud_tagspaces_sync_status](https://cloud.githubusercontent.com/assets/12627719/15966953/a7dfcd0a-2f25-11e6-8ac3-4c78457cc777.PNG)

I even installed ImageMagick and can confirm that it works from the cmd line:

![tagspaces_convert](https://cloud.githubusercontent.com/assets/12627719/15967034/048dc160-2f26-11e6-8117-e87d3fca94da.PNG)

My mitigation for now is to create the thumbnails manually by using irfanview's Batch conversion functions while setting the longest side to 400px. It also solves the orientation problem that I reported in another issue.
